vmdb/app/models/container_group.rb

```diff
@@ -9,7 +9,7 @@ class ContainerGroup < ActiveRecord::Base
   has_many :containers, :dependent => :destroy
   has_many :container_definitions, :dependent => :destroy
   belongs_to  :ext_management_system, :foreign_key => "ems_id"
-  has_many :labels, :class_name => CustomAttribute, :as => :resource, :conditions => {:section => "labels"}
+  has_many :labels, -> { where(:section => "labels") }, :class_name => CustomAttribute, :as => :resource
```

Comment:
Also I still hate Ruby 1.9 hash syntax... THERE I SAID IT :stuck_out_tongue_closed_eyes: